imaging/jasperreports/_index.md

```diff
@@ -10,7 +10,7 @@ description: Image processing and manipulation library API  to create edit draw
 
 {{< blocks/products/pf/main-container pfName="Aspose.Imaging" subTitlepfName="for JasperReports" >}}
 
-{{< blocks/products/pf/sub-menu logoImageSrc="https://www.aspose.com/templates/aspose/App_Themes/V3/images/imaging/272x272/aspose_imaging-for-jasperreports.png" liveDemosLink="" PricingLink="https://purchase.aspose.com/pricing/imaging/jasperreports" buyLink="" docsLink="https://docs.aspose.com/imaging/jasperreports/release-notes/" instalationsDocsLink="" nugetLink="" nugetPackageName="" >}}
+{{< blocks/products/pf/sub-menu logoImageSrc="https://www.aspose.com/templates/aspose/App_Themes/V3/images/imaging/272x272/aspose_imaging-for-jasperreports.png" liveDemosLink="" PricingLink="https://purchase.aspose.com/pricing/imaging/jasperreports" buyLink="https://purchase.aspose.com/pricing/imaging/jasperreports" docsLink="https://docs.aspose.com/imaging/jasperreports/release-notes/" installationsDocsLink="" nugetLink="" nugetPackageName="Raster Formats:" mavenRepoLink="" directDownloadLink="https://downloads.aspose.com/imaging/jasperreports" >}}
```

Comment:
If you add `directDownloadLink="https://downloads.aspose.com/imaging/jasperreports"` then the Download button will work as a link that redirects to the link `"https://downloads.aspose.com/imaging/jasperreports"`
`nugetPackageName="Raster Formats:"` parameter`s used in pop-up that shows up when you click on the download button. If **directDownloadLink** parameter is defined then the button will work only as a link, it will no trigger  pop-up and no one will see the text related to the NuGet. So, this parameter is an extra one.